docs/feature_flags.md

```diff
@@ -93,7 +93,4 @@ computed at all.
 
 `--enable-feature=auto-gomaxprocs`
 
-When enabled, GOMAXPROCS variable will be automatically set to match the container CPU limit.
-
-This means that Go runtime will operate as if it had only amount of CPU specified in the container
```

Comment:
I removed this wording, i can see how people allocating 3.5 CPU's to prometheus might complain that it is using 3 or 4 CPU's and not 3.5. I sticked to upstream repo's wording instead.

Reply:
Sure, NP.